testing/kuttl/e2e/pgbouncer/13--read-certificate.yaml

```diff
@@ -25,4 +25,4 @@ spec:
             - -ceu
             - |
               openssl s_client --connect '$(PGHOST):$(PGPORT)' --starttls postgres < /dev/null 2> /dev/null |
-              openssl x509 --noout --text
+              openssl x509 -noout -text
```

Comment:
🤔 `--connect` is okay on the line above, but `--text` is not?

Reply:
huh, when I was trying this with an image that had `openssl` (`1.1.1n`, as opposed to `1.1.1k`, which was what our PSQL image has), it gave an error about `--noout`, so when I checked the docs, I noticed that it had `-noout` -- but you're right, it also has `-connect`...